src/components/Input/styles.module.scss

```diff
@@ -0,0 +1,27 @@
+@import 'scss/variables/_colors';
+
```

Comment:
Why are these blank lines between each import? if you can eliminate them better.